### ISSUE TITLE
ToggleButton loses sliding animation when "included" in a layout

### ISSUE THREAD

**Issue by MewX** [zcweng/ToggleButton#2]
I created a layout to contain the ToggleButton, then I created another layout and use "include" mark to quote the layout, which contains the ToggleButton.

I ran **the app with the second layout**, and found ToggleButton had no sliding animation effects.

For example:

> First xml: layout1.xml

```xml
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:toggle="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:orientation="vertical" >

    <com.zcw.togglebutton.ToggleButton
        android:id="@+id/switcher"
        android:layout_alignParentRight="true"
        android:layout_centerVertical="true"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
    </com.zcw.togglebutton.ToggleButton>

</RelativeLayout>
```

> Second xml: layout2.xml

```xml
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:toggle="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical" >

    <include layout="@layout/layout1" />

</RelativeLayout>
```

**Comment by zcweng** [comment]
i will fix it later.thks

**Comment by zcweng** [comment]
fixed #2, add default layout bounds, width:50dp, height:30dp
